src/preProcessPattern.js

```diff
@@ -48,6 +48,11 @@ export default function preProcessPattern(globalRef, pattern) {
   pattern.context = normalizePath(pattern.context);
   pattern.ignore = globalRef.ignore.concat(pattern.ignore || []);
 
+  // One global and one pattern-specific ignore function is allowed
+  pattern._ignoreFunctions = [];
+  if (globalRef.ignoreFiles && typeof globalRef.ignoreFiles === 'function') pattern._ignoreFunctions.push(globalRef.ignoreFiles);
```

Comment:
Avoid `globalRef`, `ignore` and `context` on global level will be removed in next major release in favor `pattern.ignore` and `pattern.context`

Reply:
so there should be no global pattern possible anymore? just on the pattern directly?

Reply:
yes